packages/encoding/test/readConditions.test.ts

```diff
@@ -0,0 +1,38 @@
+import { expect } from 'earljs'
+
+import { DecodingError } from '../src'
+import { readConditions } from '../src/readConditions'
+import { ByteWriter } from './ByteWriter'
+import { readToDecode } from './readToDecode'
+
+describe(readConditions.name, () => {
+  const decode = readToDecode(readConditions)
+
+  it('fails for empty data', () => {
+    expect(() => decode('')).toThrow(DecodingError, 'Went out of bounds')
+  })
+
+  it('can read zero conditions', () => {
```

Comment:
IMO `can` word does not bring any value in test case title - if we made it `reads zero conditions` it says the same thing, but is shorter at the same time. Wdyt?

Reply:
I am happy to change it but it is already like this everywhere. If you insist we can make a big update in another PR
![image](https://user-images.githubusercontent.com/17070569/162216373-23223dfa-faec-4df7-867f-0c550bb11f69.png)

Reply:
Ok, makes sense - let's do it in another PR at some point.